kernel/apic/Cargo.toml

```diff
@@ -3,6 +3,7 @@ authors = ["Kevin Boos <kevinaboos@gmail.com>"]
 name = "apic"
 description = "APIC (Advanced Programmable Interrupt Controller) support for Theseus (x86 only), including apic/xapic and x2apic"
 version = "0.1.0"
+edition = "2018"
```

Comment:
Is there a reason we don't go straight to the `2021` edition?

Reply:
only that i haven't looked into it and don't know what `2021` includes :grimacing: